react/tic-tac-toe/src/app/screens/App/components/Results/index.js

```diff
@@ -1,7 +1,47 @@
-import React from 'react';
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import actions from 'redux/results/actions';
+import CustomTable from './components/CustomTable';
+import Button from './components/Botton';
+import styles from './styles.module.scss';
 
-function Result (props) {
-  return <h1>Comming Soon</h1>
+class Result extends Component {
+  handleOnClick = () => {
+    this.props.history.goBack();
+  }
+
+  componentDidMount() {
```

Comment:
Component's lifecycle methods should be put first in the class declaration. Check [this](https://github.com/Wolox/tech-guides/blob/master/react/style-guide.md) for more info on how to arrange them.